install/kubernetes/cilium/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 1.9.90
+version: 1.9.91
```

Comment:
can you revert this back ? version will be updated as part of release process.

Reply:
I think the file cilium-1.9.91.tgz is checked in by mistake :thinking:

Reply:
3rd time's the charm